crates/anstyle-wincon/src/windows.rs

```diff
@@ -9,19 +9,13 @@ type StdioColorInnerResult = Result<(anstyle::AnsiColor, anstyle::AnsiColor), in
 /// Cached [`get_colors`] call for [`std::io::stdout`]
 pub fn stdout_initial_colors() -> StdioColorResult {
     static INITIAL: std::sync::OnceLock<StdioColorInnerResult> = std::sync::OnceLock::new();
-    INITIAL
-        .get_or_init(|| get_colors_(&std::io::stdout()))
-        .clone()
-        .map_err(Into::into)
+    (*INITIAL.get_or_init(|| get_colors_(&std::io::stdout()))).map_err(Into::into)
 }
 
 /// Cached [`get_colors`] call for [`std::io::stderr`]
 pub fn stderr_initial_colors() -> StdioColorResult {
     static INITIAL: std::sync::OnceLock<StdioColorInnerResult> = std::sync::OnceLock::new();
-    INITIAL
-        .get_or_init(|| get_colors_(&std::io::stderr()))
-        .clone()
-        .map_err(Into::into)
+    (*INITIAL.get_or_init(|| get_colors_(&std::io::stderr()))).map_err(Into::into)
```

Comment:
I applied the suggestion for `clippy::clone_on_copy` but I was not sure what to do about the other warning about `OnceLock` being too new for the MSRV:

```
warning: current MSRV (Minimum Supported Rust Version) is `1.66.0` but this item is stable since `1.70.0`
  --> crates\anstyle-wincon\src\windows.rs:13:10
   |
13 |         .get_or_init(|| get_colors_(&std::io::stdout()))
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#incompatible_msrv
   = note: `#[warn(clippy::incompatible_msrv)]` on by default

```